### ISSUE TITLE
Traceback Error - even when uninstalling and reinstalling

### ISSUE THREAD

**Issue by Curiorodent** [nianeyna/ao3downloader#168]
Tried running it after using gallery-dl to download a bunch of twitter stuff (gallery-dl still works). When I opened the program, I got this wall of test; 

Traceback (most recent call last):
  File "B:\ao3-script\ao3downloader-main\venv\Lib\site-packages\pdfquery\pdfquery.py", line 20, in <module>
    from pdfminer.pdfparser import PDFDocument, PDFPage
ImportError: cannot import name 'PDFDocument' from 'pdfminer.pdfparser' (B:\ao3-script\ao3downloader-main\venv\Lib\site-packages\pdfminer\pdfparser.py)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "B:\ao3-script\ao3downloader-main\ao3downloader.py", line 7, in <module>
    from ao3downloader.actions import updatefics
  File "B:\ao3-script\ao3downloader-main\ao3downloader\actions\updatefics.py", line 4, in <module>
    from ao3downloader import parse_text, strings, update
  File "B:\ao3-script\ao3downloader-main\ao3downloader\update.py", line 7, in <module>
    import pdfquery
  File "B:\ao3-script\ao3downloader-main\venv\Lib\site-packages\pdfquery\__init__.py", line 1, in <module>
    from .pdfquery import PDFQuery
  File "B:\ao3-script\ao3downloader-main\venv\Lib\site-packages\pdfquery\pdfquery.py", line 23, in <module>
    from pdfminer.pdfdocument import PDFDocument
  File "B:\ao3-script\ao3downloader-main\venv\Lib\site-packages\pdfminer\pdfdocument.py", line 23, in <module>
    from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
  File "B:\ao3-script\ao3downloader-main\venv\Lib\site-packages\cryptography\hazmat\primitives\ciphers\__init__.py", line 11, in <module>
    from cryptography.hazmat.primitives.ciphers.base import (
  File "B:\ao3-script\ao3downloader-main\venv\Lib\site-packages\cryptography\hazmat\primitives\ciphers\base.py", line 10, in <module>
    from cryptography.hazmat.bindings._rust import openssl as rust_openssl
ImportError: DLL load failed while importing _rust: The specified module could not be found.
Press any key to continue . . .

I deleted everything, reinstalled it, and the same error happened. It uninstalled pip and then reinstalled a newer (?) version.

**Comment by nianeyna** [comment]
Closing due to inactivity

**Comment by Leedie** [comment]
I had the same issue, but fixed it!

What I tried:
1. Installing Python 3.11.4 and trying to run the program. Received following error:
ModuleNotFoundError: No module named 'requests'
2. Unintalled Python 3.11.4, Installed Python 3.9.0 (made sure to check "Add Python to environment variables"). Also redownloaded the repository, made sure I'm clicking file in actual unzipped folder, then changed app execution aliases. Received new error, word for word as in the first message (only folder path was different, mine is C:\Users\Leedie\Desktop\ao3downloader-main)
3. Tried reinstalling Python 3.9.0 (made sure to check "Add Python to environment variables"), rebooting the computer, redownloading repository, deleting venv, checking app execution aliases. Could not do the following: "If you are able to create logvisualization.html (menu option 'v'), take a look through the logs to see if there are any helpful error messages." as pushing any button in the command line window closes that window (last line "Press any key to continue . . ."). Received the same error :(
Copy-pasted the command line text (with Ctrl+Shift+C) into a manually created log.txt file
[Log.txt](https://github.com/user-attachments/files/18552105/Log.txt)
I also found [this](https://stackoverflow.com/questions/78853596/how-to-solve-importerror-dll-load-failed-while-importing-rust) stackoverflow for seemingly same error and manually updated the cryptography (to version 41.0.7) and bcrypt (to version 3.2.0) packages, issue persisted on Python 3.9.0

After that I uninstalled Python 3.9.0 and installed Python 3.11.4 and redownloaded ao3downloader from GitHub and it worked!

So if anyone else is having the same issue my advice would be to try reinstalling Python to the newer version after going through all Troubleshooting steps, and maybe manually update cryptography (to version 41.0.7) and bcrypt (to version 3.2.0) and then try reinstalling Python/redownloading ao3downloader if that does not work.
